include/complex_quda.h

```diff
@@ -1207,7 +1207,7 @@ lhs.real()*rhs.imag()+lhs.imag()*rhs.real());
   template <typename real> __host__ __device__ inline complex<real> i_(const complex<real> &a)
   {
     // FIXME compiler generates worse code with "optimal" code
-#if 1
```

Comment:
Why did you make this change?

Reply:
Sure, the assembly can be inspected using cuobjdump
```
cuobjdump --dump-sass file.o
```
will dump the GPU assembly to stdout.  There's a variety of options for `cuobjdump` that can be inspected with `--help`, e.g., to limit which kernels are dumped.

Another interesting option is `--dump-resource-usage` which will just dump a summary of the resource usage for each kernel (registers, shared memory, stack frame, etc.)